src/corehost/cli/deps_resolver.cpp

```diff
@@ -583,9 +615,7 @@ bool deps_resolver_t::resolve_probe_dirs(
             // because of rid-fallback could happen (ex: CentOS falling back to RHEL)
             if ((entry.asset_name == _X("apphost")) && ends_with(entry.library_name, _X(".Microsoft.NETCore.DotNetAppHost"), false))
             {
-                trace::warning(_X("Warning: assembly specified in the dependencies manifest was not found -- package: '%s', version: '%s', path: '%s'"), 
-                    entry.library_name.c_str(), entry.library_version.c_str(), entry.relative_path.c_str());
-                return true;
+                return report_missing_assembly_in_manifest(entry, true);
```

Comment:
Do you want to log an issue for this, or change it now? If we return false, this condition will cause the app not to load. A case of misconfigured deps data? Looking at history, it's been this way (returning true) since the code was added in May 2016.